projects/angular/src/accordion/_accordion.clarity.scss

```diff
@@ -581,22 +306,55 @@
     }
 
     .clr-accordion-status {
-      min-width: variables.$clr_baselineRem_3;
-      @include mixins.css-var(
-        margin-left,
-        clr-accordion-step-status-margin-left,
-        accordion-variables.$clr-accordion-step-status-margin-left,
-        variables.$clr-use-custom-properties
-      );
+      min-width: tokens.$cds-global-space-14;
+      margin-left: accordion-variables.$clr-accordion-step-status-margin-left;
     }
 
     .clr-accordion-number {
+      @include mixins.generate-typography-token('BODY-14-SB-STD');
+      padding: 0 tokens.$cds-global-space-7;
+      display: inline-block;
+    }
+
+    .clr-accordion-description {
       display: inline-block;
+      max-width: mixins.baselinePx(650);
+
+      @media (min-width: accordion-variables.$clr-accordion-responsive-breakpoint) {
+        margin-left: tokens.$cds-global-space-12;
+      }
+    }
```

Comment:
In figma it seems that the description is all the way in the right? Am I missing something?